### PR TITLE
New roles

### PR DESCRIPTION
**Pull request by northofnormal**
set platform-specific groups and roles

### CODE REVIEW

File: scripts/roles.coffee [hunk 1]
```diff
@@ -18,6 +18,10 @@
 #   hubot roles people rm <names>- Remove people
 #   hubot roles people set <people> - Set all people at once
 #   hubot roles shuffle - Randomly assign people to roles
+#   hubot roles android - List the people in android
+#   hubot roles ios - List the people in ios
+#   hubot roles android add <names> - Add people to android
+#   hubot roles ios add <names> - Add people to iOS
```
Comment (nwest) [inline scripts/roles.coffee]: Looks like roles android rm and roles ios rm got missed here.

File: scripts/roles.coffee [hunk 2]
```diff
@@ -74,11 +84,18 @@ module.exports = (robot) ->
...
 
     if roles.length > people.length
       msg.send "Not enough people to cover all roles"
     else
       newRoles = _.object(roles, _.sample(people, roles.length))
+      newRoles = _.omit(newRoles, "🔱  Android Alchemist", "✨  iOS Illusionist")
+      newAlchemist = _.object(["🔱  Android Alchemist"], _.sample(android, 1))
+      newIllusionist = _.object(["✨  iOS Illusionist"], _.sample(ios, 1))
+
+      _.extend(newRoles, newAlchemist, newIllusionist)
```
Comment (nwest) [inline scripts/roles.coffee]: Hm. if other rooms are using this, it kind of forces them to use these roles in addition to what they add. Thoughts?

Comment (northofnormal) [inline scripts/roles.coffee]: I had the same thought this morning, and I've got an idea on a fix. Hold, please.

File: scripts/roles.coffee [hunk 3]
```diff
@@ -18,6 +18,12 @@
 #   hubot roles people rm <names>- Remove people
 #   hubot roles people set <people> - Set all people at once
 #   hubot roles shuffle - Randomly assign people to roles
+#   hubot roles android - List the people in android
+#   hubot roles android add <names> - Add people to android
+#   hubot roles android rm <names> - Remove people from android
+#   hubot roles ios - List the people in ios
+#   hubot roles ios add <names> - Add people to iOS
+#   hubot roles ios rm <names> - Removie people from iOS
```
Comment (nwest) [inline scripts/roles.coffee]: "Removie"

**Pull request merged**